.github/workflows/pull_request.yml

```diff
@@ -32,10 +34,26 @@ jobs:
           args: -d -z
 
       - uses: actions/upload-artifact@v4
+        id: upload-artifact
         with:
           name: WeakAuras-PR#${{ github.event.number }}-${{ github.sha }}
           path: .release/
 
+      - name: test
```

Comment:
what does this do exactly?

Reply:
anything you want, so long as that isn't very useful

Reply:
-
![ConfusedConfusedFaceGIF](https://github.com/WeakAuras/WeakAuras2/assets/75278/ddc77eef-81a8-45af-a536-19f48f87b641)

Reply:
It obviously tests duh